radicle-cli/src/commands/patch.rs

```diff
@@ -19,7 +19,7 @@ use crate::terminal::patch::Comment;
 
 pub const HELP: Help = Help {
     name: "patch",
-    description: "Work with radicle patches",
+    description: "Work with a patch",
```

Comment:
Maybe "manage patches"?  "work with a patch" excludes creating one, is this the case?